---
layout: archive
title: "Projects"
permalink: /projects/
author_profile: true
redirect_from:
  - /resume
---

ERC Starting Grant SUMOwriteNread (01/2023-12/2027)
------

The "SUMOwriteNread" project, dedicated to mechanisms of protein SUMOylation and its functional consequences, currently involves PhD students, Lucija Mance and Aanchal Mishra, a postdoc, El hadji Cisse, permanent engineers, Stéphane Goffinont and Franck Coste, and myself as a supervisor. We receive a lot of help from other group members and other scientists from the CBM and beyond.

What is protein SUMOylation? Small ubiquitin-like modifiers (SUMOs), discovered about 25 years ago, are members of the superfamily of ubiquitin-like proteins. We speak of protein SUMOylation when these proteins become covalently attached to intracellular target proteins, modifying their function, location or stability. SUMOylation plays an essential role in many biological functions and has strong links to disease, especially cancer. SUMOylation typically depends on enzymes called E3 ligases. Few SUMO E3 enzymes have been identified and characterised. The ERC-funded SUMOwriteNread project will fill this gap, identifying the structure and mechanism of action of some SUMO E3 ligases, developing approaches to identify and characterise new SUMO E3s, investigate mechanisms by which the specificity of SUMOylation is achieved, and finally to investigate the functional impact of SUMOylation in terms of triggering new protein:protein interactions.

The project is funded by a European Research Council (ERC) Starting Grant (ID 101078837) from the European Union, as part of the Horizon Europe programme. It is active from January 2023 until December 2027. Our budget amounts to € 1 493 515.

<img src="https://msuskiewicz.github.io/images/LOGO_ERC-FLAG_FP.png" width='300' />

HFSP Early Career TFilament (09/2025-08/2028)
------

Within the framework of the SUMOWriteNread described above, we studied the SUMOylation of proteins from the ZBTB transcription factor family, accidentally discovering that several proteins from this family have a tendency to form protein filaments. This prompted us to ask: how common is filament formation in other transcription factor families? Is it a frequent, but overlooked, feature of proteins involved in transcriptional regulation? DNA is inherently an elongated, repetitive molecule, and motifs engaging transcription factors are known to be often repeated in tandem; it would therefore seem logical that proteins that recognise DNA to regulate genes could also frequently form elongated, repetitive structures. To make the long story short, these ideas have matured into a project, which we constructed together with two collaborators: <a href="https://www.stallerlab.com/">Max Staller from Berkeley</a>, who studied gene regulation in cells using a combination of high-throughput assays and sophisticated data analysis, and <a href="">Antoni Wróbel at Oxford</a>, an experienced structural biologist mainly specialised in viral proteins. We are about to launch this collaborative project on 1st September 2025. The project was selected for funding by Human Frontiers Science Program (HFSP) an organisation supporting frontier research. Better known for postdoctoral fellowships, HFSP also finances a few dozen research projects every year, with an emphasis on "intercontinental collaborations". The total funding amounts to 1.2 million American dollars.

Here follows a brief, simplified description of the project: The regulation of gene expression is essential for development,stressresponses, and homeostasis. During this process, the information encoded in the genome isturned into functional proteins. Transcription of a deoxyribonucleic acid (DNA) segment creates a ribonucleic acid (RNA) molecule, which acts as a blueprint to make a protein, a functional component of the cell. The transcription of genesinto RNA isregulated by proteins called transcription factors. Transcription factors have two main functions: they bind to specific DNA sequences and either activate or represstranscription. Traditionally, it was believed that transcription factors worked alone or in pairs. However, recent research has unveiled a surprising twist: many transcription factors often join forces, forming larger assemblies of moleculesthat together navigate DNA to identify specific regions. The nature of these assembliesis poorly understood. Our project focuses on one particular category of these assemblies called filaments. Filaments are open-ended, rod-like structures composed of multiple copies of the same protein. Based on our preliminary analysis, we propose that many human transcription factors might have unreported abilitiesto form filaments, which would allow them to simultaneously bind to long DNA sequences. To study how filament formation affectsthe function of transcription factors and how general this mechanism is, we will rely on various approaches. First, we will use experiments, computational analysis, and machine learning to study known examples of filamentation. We will also use cryo-electron microscopy, an imaging technique suited for tiny objects, to study structures of filamentsformed by transcription factorsin the presence or absence of DNA. Finally, we will employ a transformative software called AlphaFold to predict new human transcription factors with a propensity to form filaments. Overall, the project will establish the prevalence and importance of filamentation as a mechanism by which transcription factorsregulate expression of human genes.



In addition to these main funded projects, we have several other active small projects related to protein modifications and protein interactions.


